odata-vocabularies/meta.yaml

```
@@ -0,0 +1,17 @@
+pagetitle: OData Vocabularies Version 4.0
+subtitle: Committee Specification Draft 02 / Public Review Draft 02
```

```suggestion
subtitle: Committee Specification Draft 02
```
Simplified terminology, see title pages of our 4.02 CSD01 docs